Tasks/XamarinTestCloud/xamarintestcloud.ts

```diff
@@ -229,7 +229,10 @@ var submitToTestCloud = function (index) {
     // For an iOS .ipa app, look for an accompanying dSYM file
     if (dsym && path.extname(appFiles[index]) == '.ipa') {
         // Find dSYM files matching the specified pattern
-        var alldsymFiles = tl.find(path.dirname(appFiles[index]));
+        // Check in one folder up since IPAs are now generated under a timestamped folder
+        var ipaFolder = path.dirname(path.dirname(appFiles[index]));
```

Comment:
@yacaovsnc: It shouldn't cause any compat issue since we will just be looking for .dSYM files one level up. Since the Xamarin support for Xcode 7 the IPAs are generated in a folder under the outDir where as the .dSYM file is under the outDir. So the task hasn't been finding the .dSYM files.